### DISCUSSION TITLE
Code Editor slow load

### DISCUSSION THREAD

**Discussion by Saldef** [standardnotes/forum#2926] [General]
**Describe the bug**
Code Editor takes around 10-15 seconds to load 

**To Reproduce**
Open any note with Code Editor

**Expected behavior**
Loads a bit faster 

**Desktop**
 - OS: Windows 10 20h2
 - App Version: 3.132.2

**Comment by myreli** [comment]
Hi @Saldef, may I ask what size is your note? 

Do you also experience this on the web app or the latest version (`3.150.0`)?
